vendor/k8s.io/kubernetes/pkg/api/validation/validation.go

```diff
@@ -2544,6 +2544,17 @@ func ValidateLimitRange(limitRange *api.LimitRange) field.ErrorList {
 			}
 		}
 
+		if limit.Type == api.LimitTypePersistentVolumeClaim {
+			_, minQuantityFound := limit.Min[api.ResourceStorage]
```

Comment:
this is a confusing special case and I would prefer to get rid of it.

can we get rid of it here and make a separate pr to upstream that i can merge?